invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -142,6 +143,10 @@ export class RDMDepositForm extends Component {
         files={this.props.files}
         permissions={this.props.permissions}
       >
+        <CommunityHeader
+          record={this.props.record}
```

Comment:
why not using prop deconstruction ? prettier didn't complain about this?
`const {record} = this.props`;
I would also refactor the other variables too